docs/api/browser-window.md

```diff
@@ -398,6 +398,10 @@ It creates a new `BrowserWindow` with native properties as set by the `options`.
       * `bypassHeatCheck` - Bypass code caching heuristics but with lazy compilation
       * `bypassHeatCheckAndEagerCompile` - Same as above except compilation is eager.
       Default policy is `code`.
+    * `preferredSizeMode` Boolean (optional) - Whether to enable preferred size
```

Comment:
Should document the default value.

Also I think there should be some descriptions on what "preferred size" is and what does it do, it is not very clear from current documents.

Reply:
How's this? 97148e8

Reply:
Added one minor improvement in 1daae04